install/install.md

```diff
@@ -112,8 +123,11 @@ All provisioned systems need to be RAPIDS capable. Here's what is required:
 - <i class="fas fa-check-circle"></i> [CUDA 11.4](https://developer.nvidia.com/cuda-11-4-0-download-archive){: target="_blank"} with Driver 470.42.01 or newer
 - <i class="fas fa-check-circle"></i> [CUDA 11.5](https://developer.nvidia.com/cuda-11-5-0-download-archive){: target="_blank"} with Driver 495.29.05 or newer
 - <i class="fas fa-check-circle"></i> [CUDA 11.8](https://developer.nvidia.com/cuda-11-8-0-download-archive){: target="_blank"} with Driver 520.61.05 or newer
+- <i class="fas fa-check-circle"></i> [CUDA 12.0](https://developer.nvidia.com/cuda-11-8-0-download-archive){: target="_blank"} with Driver 525.60.13 or newer **for pip installations only**
 
- **Note**: RAPIDS is tested with and officially supports the versions listed above. Newer CUDA and driver versions may also work with RAPIDS. See [CUDA compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html) for details.
+ **Note**: RAPIDS is tested with and officially supports the versions listed above. Newer CUDA and driver versions may also work with RAPIDS. See [CUDA compatibility](https://docs.nvidia.com/deploy/cuda-compatibility/index.html) for details. 
+ 
+ **Note**: Docker and Conda CUDA 11.x installations function as expected on a CUDA 12.x machine. pip installations require using the matching wheel to the system's installed CUDA. For CUDA 11.x machines, install the <code>-cu11</code> wheels, and for CUDA 12.x machines install the <code>-cu12</code> wheels.
```

Comment:
What does "CUDA 12.x machine" mean? We need to speak in terms of **driver** or **toolkit**. "Machine" is too ambiguous.

Reply:
Agreed! 

>**Note**: RAPIDS Docker and Conda CUDA 11.x environments function as expected on a system with a CUDA 12.x driver and they include their own toolkit pieces. <br>pip installations require using the matching wheel to the system's installed CUDA toolkit. For CUDA 11.x toolkits, install the <code>-cu11</code> wheels, and for CUDA 12.x tookits install the <code>-cu12</code> wheels. If your installation has a CUDA 12.x driver but a CUDA 11.x toolkit, use the <code>-cu11</code> wheels.

?

Reply:
> and they include their own toolkit pieces.

change to
> because they include their own toolkit.

Reply:
I've made these changes 🎉